event/event.go

```diff
@@ -421,3 +423,19 @@ func getNamespaceFromEvent(event *api.Event) string {
 
 	return ""
 }
+
+// getCacheVolumeFromEvent gets the cache volume from event data for worker. Will return empty string if can not get it.
+func getCacheVolumeFromEvent(event *api.Event) string {
+	if event == nil {
+		log.Error("Can not get cache volume from data as event is nil")
+		return ""
+	}
+
+	if v, e := event.Data["cachevolume"]; e {
```

Comment:
maybe the map key should be variable?

Reply:
This PR is just for `release-0.3`, it will not be like this in the latest implementation.

Reply:
Done